src/test_driver/openiotsdk/integration-tests/shell/test_app.py

```diff
@@ -91,6 +94,9 @@ def test_command_check(device):
     ret = device.wait_for_output("Open IoT SDK shell example application run")
     assert ret is not None and len(ret) > 0
 
+    # Wait for printing prompt ">", we can't wait for it because it is not terminated by a newline character
+    time.sleep(1)
```

Comment:
This is a sleep in a test ... generally frowned upon. Is there no way to make the I/O non-line-buffered to avoid that? Maybe add a TODO for this. time.sleep is not ideal (why is 1s enough? why not 0.5 or 10?)

Reply:
I changed the shell example prompt to a message that ended with a newline character. This avoids sleep usage and guarantees the correct parsing of application output.